packages/rancher-cis-benchmark/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: local
-packageVersion: 01
-releaseCandidateVersion: 01
+packageVersion: 00
```

Comment:
packageVersion also decreases to 00?

Reply:
I dropped the packageVersion to 00 in favor of just bumping the Chart version from 1.0.3 to 1.0.4.

packageVersion == rancherPatchVersion, so for local charts (where we have control over the patch number) it just makes more sense to bump the real patch version itself.